app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -40,7 +40,8 @@ def destroy
 
     def apply_secure_headers_override
       override_content_security_policy_directives(
-        form_action: ["'self'", @authorize_form.allowed_form_action].compact
+        form_action: ["'self'", @authorize_form.allowed_form_action].compact,
+        preserve_schemes: true
```

Comment:
This will mean that HTTP will be allowed in production too, which we don't want. How about wrapping this inside a conditional, like we do here: https://github.com/18F/identity-idp/blob/master/config/initializers/secure_headers.rb#L31-L38
For example, you can update that conditional to set `preserve_schemes` to `true` when not in production:
```ruby
if !Rails.env.production?
  config.csp = default_csp_config.merge(
    script_src: ["'self'", "'unsafe-eval'", "'unsafe-inline'"],
    style_src: ["'self'", "'unsafe-inline'"],
    preserve_schemes: true
  )
else
  config.csp = default_csp_config
end
```